### PR TITLE
Audio fixes and enhancements

### LINKED ISSUE #236: Asset pools not getting created
Both video pools and sound pools are not getting created, however image pools are working just fine (when all are specified in a configuration file).

### PR DESCRIPTION
**Pull request by qcapen**
Fixed #236 (Asset pool loading bug), added parameters to sound_player in config_validator: action, events_when_played, events_when_stopped

### CODE REVIEW

File: mpf/core/config_validator.py [hunk 1]
```diff
@@ -820,8 +821,6 @@
     type: single|enum(NC,NO)|NO
     debounce: single|enum(auto,quick,normal)|auto
     ignore_window_ms: single|ms|0
-    events_when_activated: list|str|None
```
Comment (jabdoa2) [inline mpf/core/config_validator.py]: Why did you remove those? Breaks the tests

**Comment by qcapen** [comment]
That change to config_validator happened somehow in the merge process.  I certainly did not intend for that line to be removed.  I'll fix it.
